### PR TITLE
add taskr 0.3.1

### PR DESCRIPTION
**Pull request by exherb**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/taskr.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'taskr' do
+  version '0.2.2'
+  sha256 '56c92a194911ab58b969d3a91ef7d5b454b8d7dc254e9a9cb189ea48862e68c0'
+
+  # github.com was verified as official when first introduced to the cask
```
Comment (commitay) [inline Casks/taskr.rb]: `github.com/bukinoshita/taskr`

File: Casks/taskr.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  url "https://github.com/bukinoshita/taskr/releases/download/#{version}/taskr-#{version}-mac.zip"
+  name 'Taskr'
```
Comment (commitay) [inline Casks/taskr.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)

Example:
https://github.com/caskroom/homebrew-cask/blob/0480e1f119f2742586f35b65a4efbfba670757b8/Casks/autodmg.rb#L5-L8

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/taskr.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask 'taskr' do
+  version '0.2.2'
```
Comment (commitay) [inline Casks/taskr.rb]: Current version is `0.3.1`

https://github.com/bukinoshita/taskr/releases/tag/0.3.1

**Comment by exherb** [comment]
sorry, I should check this all before submit a pr. (looks like the official site is not the latest version.)

**Comment by commitay** [comment]
Just for future reference we prefer that contributors don't fixup/squash commits, it makes it easier to check what changes have been made.

**Comment by exherb** [comment]
what happened to CI?

**Comment by commitay** [comment]
```
>>> brew cask style Casks/taskr.rb
== Casks/taskr.rb ==
C:  7: 64: Trailing whitespace detected.
C:  8: 10: Align the parameters of a method call if they span more than one line.
C:  8: 88: Trailing whitespace detected.

1 file inspected, 3 offenses detected
Error: style check failed
```

**Review by commitay — APPROVED** [review]

**Comment by Jafnee** [comment]
https://taskr.now.sh/ is the official site according to the desc on https://github.com/bukinoshita/taskr , instead of https://gettaskr.now.sh/

**Review by commitay — APPROVED** [review]

**Pull request merged**